### PR TITLE
httpserver tests (#452)

### PR DESCRIPTION
**Pull request by grob**
This pull requests adds tests for `ringo/httpserver` module, and also limits logging in tests to level `error` and above.

### CODE REVIEW

File: modules/ringo/httpserver/httpserver.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ const HttpServer = module.exports = function HttpServer(options) {
         } else if (typeof(options) === "object" && options.constructor === Object) {
             jetty.setStopAtShutdown(options.stopAtShutdown !== false);
             jetty.setStopTimeout(options.stopTimeout || 1000);
-            jetty.setDumpAfterStart(options.dumpBeforeStart === true);
+            jetty.setDumpAfterStart(options.dumpAfterStart === true);
```
Comment (botic) [inline modules/ringo/httpserver/httpserver.js:69]: Nice finding!

**Review by botic — COMMENTED** [review]
Should I merge and we will look if it works on Github Actions now? Or have you found the weak spot of the tests which prevented successful runs in CI?

File: test/all.js [hunk 2]
```diff
@@ -26,13 +27,6 @@ exports.testIo             = require('./io_test');
 exports.testModules        = require('./modules/all');
 exports.testRhino          = require("./rhino/all");
 
-// Test only executed outside of Github Actions:
-if (system.env["GITHUB_ACTIONS"] === undefined) {
-    exports.testWebSocket = require('./ringo/websocket_test');
```
[the remark this answers is not in the document]
Comment (grob) [reply test/all.js:31]: no, i have no idea how i could test that – any hints?. the lines deleted above are now in test/ringo/httpserver/all.js

[the remark this answers is not in the document]
Comment (botic) [reply test/all.js:31]: I will merge and we will see 😃 Maye use a cleaner way to skip this, e.g. providing a generic `SKIP_IN_CI` variable.

**Pull request merged**